pydy/viz/scene.py

```diff
@@ -752,16 +752,37 @@ def display_ipython(self):
             self._initial_conditions_text_widgets = OrderedDict()
             self._fill_initial_conditions_widgets()
             title = widgets.HTML('<h4 style="margin-left: 5ex;">Initial Conditions</h4>')
-            self._initial_conditions_container.children = ((title, ) +
-                 tuple(v for v in self._initial_conditions_text_widgets.values()))
+            if len(self._initial_conditions_text_widgets) > 5:
```

Comment:
@oliverlee i have another idea. we display first five variable and put another variable in dropdpwn.